src/components/tools/tool-tile.tsx

```diff
@@ -197,6 +218,11 @@ export class ToolTileComponent extends BaseComponent<IProps, IState> {
                             <DragTileButton divRef={elt => this.dragElement = elt}
                               hovered={hoverTile} selected={isTileSelected}
                               onClick={e => ui.setSelectedTile(model, {append: hasSelectionModifier(e)})} />;
+    const resizeTileButton = <ResizeTileButton divRef={elt => this.resizeElement = elt}
```

Comment:
I would test `isUserResizable` here (as is done with `isDraggable` above) rather than below so that we don't even render the button internally if we don't need to.